src/test/fetch.test.mts

```diff
@@ -40,8 +40,8 @@ describe('fetch', () => {
       throw Error('Should fail without valid query');
     } catch (error) {
       const { status, statusText } = error as ApiError;
-      expect(status).toBe(400);
-      expect(statusText).toBe('Bad Request');
+      expect(status).toBe(401);
```

Comment:
401 instead of 400 is now also happening on main, not related :shrug: it seems reasonable though